plugins/module_utils/redfish_utils.py

```diff
@@ -819,7 +819,10 @@ def get_disk_inventory(self, systems_uri):
                             for property in properties:
                                 if property in data:
                                     if data[property] is not None:
-                                        drive_result[property] = data[property]
+                                        if property == "Links" and "Volumes" in data["Links"].keys():
+                                            drive_result["Volumes"] = data["Links"]["Volumes"]
```

Comment:
Could we pull out `@odata.id` from the structure? I think it would be easier for the user to consume data if "Volumes" is an array of strings instead of an array of objects (with `@odata.id` as the sole key in each object).

Reply:
Hmmm, I see your point. Changed as suggested.

Reply:
Resolving this conversation. Thanks!